includes/class-ed-solr-search.php

```diff
@@ -99,9 +99,7 @@ private function do_search() {
 	 * @return string
 	 */
 	private function get_query_string() {
-		$query_string  = 'blogId:(' . implode( ' OR ', $this->blog_ids ) . ')';
-		$query_string .= ' AND postTitle:(' . $this->keywords . ')';
-		$query_string .= ' OR postContent:(' . $this->keywords . ')';
+		$query_string = 'blogId:(' . implode( ' OR ', $this->blog_ids ) . ') AND (postTitle:(' . $this->keywords . ') OR postContent:(' . $this->keywords . '))';
```

Comment:
Hey Richard. Just wanted to check if _blogId_ search should be in here? I don't think users would know the blog ID so I think we should only be searching by content and title. Sorry if I'm misreading this.

Reply:
We pull back all the blogs that someone can access - which handles private blogs. So we only search the blogs that they have permissions to see depending if they are logged in or not. That info isn't passed by the user. We do it for them. Hope that makes sense.